server/src/internalClusterTest/java/org/elasticsearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -767,8 +768,18 @@ public void testQueuedOperationsAndBrokenRepoOnMasterFailOver() throws Exception
         internalCluster().stopCurrentMasterNode();
         ensureStableCluster(3);
 
-        awaitNoMoreRunningOperations();
```

Comment:
Do we still need this statement?

Reply:
Um thanks, not sure where that went! Probably not necessary indeed since we are waiting (indefiniteyl!) on `deleteFuture::get` either way but I did not mean to remove it.